uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -203,4 +208,15 @@ private static function isValidDateAndMonth($carry_forward_expiration_date) {
 
     return true;
   }
+
+  private static function getMaxWeight() {
+    $tableName = self::getTableName();
+    $query = "SELECT MAX(weight) as max_weight FROM $tableName";
```

Comment:
as far as i know , civicrm standard is to put variables inside strings between brackets ...   something like"SELECT MAX(weight) as max_weight FROM {$tableName}" ... and while we don't follow a specific code standard yet , i think it is a good idea to follow civicrm core standard as much as possible specially for new extensions